### ISSUE TITLE
LabCompass no longer working in Ritual League

### ISSUE THREAD

**Issue by mrk-han** [yznpku/LabCompass#66]
There have been a few more reports on the Discord, but the LabCompass is no longer working in Ritual for the past 2-3 weeks

@yznpku Is there a way for me to see debug output?

**Comment by mrk-han** [comment]
![image](https://user-images.githubusercontent.com/22263679/107267212-919fef00-6a14-11eb-9b6b-f6c9d6522612.png)

**Comment by mrk-han** [comment]
![image](https://user-images.githubusercontent.com/22263679/107267232-98c6fd00-6a14-11eb-9774-94ac1b1e77f6.png)

**Comment by mrk-han** [comment]
![image](https://user-images.githubusercontent.com/22263679/107267312-b5fbcb80-6a14-11eb-8909-f377e12ca066.png)

**Comment by mrk-han** [comment]
Tried resetting my `client.txt` and enabled multi-client support, then double checked my game client path -- still unable to get it working

**Comment by platun4** [comment]
Same issue, I though it was my end problem after I moved game, but triple checked everything, should be working. Tested both standalone and steam clients, with and without administer, Compass seeing entering plaza fine, as soon enter any of labs, it goes blank for rest of run.
Found solution that **fixed** it for me, it was "**Output Dialogue To Chat**" option under **UI options**, it has to be on for overlay to read zone changes.
